transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/store/ray_store.py

```diff
@@ -0,0 +1,105 @@
+import random
+
+import ray
+from data_processing_ray.runtime.ray import RayUtils
+from ray.util import ActorPool
+
+
+@ray.remote(scheduling_strategy="SPREAD")
+class KeyedValueListActor:
+    """
+    This class uses ray actor to serve as a dictionary of type `dict[str, List[str]]`
+    and provides methods to add/get items to/from it.
+
+    """
+
+    dict_: dict
+
+    def __init__(self, a):
+        self.dict_ = {}
+
+    def put(self, key, value):
+        try:
+            self.dict_[key].append(value)
+        except KeyError:
+            self.dict_[key] = [value]
+        except AttributeError:
+            self.dict_ = {}
+            self.dict_[key] = [value]
+
+    def get(self, key):
+        try:
+            return self.dict_.get(key)
+        except:
+            return None
+
+    def keys(self, x):
+        try:
+            return self.dict_.keys()
+        except:
+            return []
+
+
+class KeyedValueListActorPool:
```

Comment:
This is not really good. Take a look at the exact dedup for the same functionality